modules/ROOT/pages/salesforce/salesforce-connector-100.adoc

```diff
@@ -0,0 +1,204 @@
+|xref:salesforce/salesforce-connector-100-config-topics.adoc#leadconvert[Lead Convert Request ID] |Specify a lead ID using DataWeave
+|xref:salesforce/salesforce-connector-examples.adoc#parentchild[Parent Child Relationships] |Create and update parent and child relationships example
+|xref:salesforce/salesforce-connector-100-config-topics.adoc#streamsub[Streaming Channel] |Subscribe to a Salesforce streaming channel
+|<<Anypoint Exchange Templates and Examples>> |Simplify Salesforce Connector development with a template or an example
+|xref:salesforce/salesforce-connector-100-config-topics.adoc#eventsandtopics[Topics] |Subscribe to a Salesforce topic
+|===
+
+[[developapp]]
+== Develop an Application
+
+Build an app in this order:
+
+. Configure the connector.
+. Test the connection.
+. Build the rest of your flow.
+. Add and configure DataWeave.
+
+When developing an app, the Salesforce Connector provides these integration patterns
+you can use with the Salesforce APIs:
+
+* Batch Data Synchronization
++
+An external system accesses, changes, deletes, or adds data in Salesforce in batches, and vice versa (Salesforce to external system).
+* Remote Call-In
++
+An external system accesses, changes, deletes, or adds data in Salesforce, and vice versa (Salesforce to external system).
+* Fire and Forget Remote Process Invocation
++
+Salesforce initiates a process in a third-party system and receives an acknowledgment that the process has started. The third-party system continues processing independent of Salesforce.
+* Request-Reply Remote Process Invocation
++
+Salesforce initiates a process in a remote system, waits for the remote system to finish processing, and accepts control back again from the remote system.
+* User Interface Update Based on Data Changes
++
+The Salesforce UI updates in response to a change in a third-party system.
+
+[[apis]]
+== Salesforce APIs
+
+Salesforce Connector works with the Salesforce SOAP API, REST API, Bulk API, and Streaming API, but does not provide access to the Chatter API or the Tooling API. Each API call uses a request/response pattern over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
+
+* Apex SOAP API
++
+Salesforce https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_api.htm?search_text=soap[Apex SOAP API] exposes Apex class methods as custom SOAP web service calls. This allows an external app to invoke an Apex web service to perform an action in Salesforce.
++
+* Apex REST API
++
+Salesforce https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_rest.htm[Apex REST API] creates your own REST-based web services using Apex. This API has all of the advantages of the REST architecture, while adding the ability to define custom logic and including automatic argument or object mapping.
```

Comment:
"adding the ability to define" > "enabling you to define"?

Reply:
Leaving as is because the ability is additional to being a REST API.